doc/libf3d/OPTIONS.md

```diff
@@ -63,6 +63,7 @@ render.line-width|double<br>1.0<br>render|Set the *width* of lines when showing
 render.show-edges|bool<br>false<br>render|Show the *cell edges*|\-\-edges
 render.point-size|double<br>10.0<br>render|Set the *size* of points when showing vertices and point sprites.|\-\-point-size
 render.grid.enable|bool<br>false<br>render|Show *a grid* aligned with the horizontal (orthogonal to the Up direction) plane.|\-\-grid
+render.grid.absolute|bool<br>false<br>render|Show the grid at the *absolute origin* of the model's coordinate system.|\-\-grid
```

Comment:
Position the grid at the *absolute origin* of the model's coordinate system instead of below the model.